### PR TITLE
Job scheduler

### PR DESCRIPTION
**Pull request by slotaj**
@bethsecor 

I combined the rake tasks to be part of one file, please check to make sure that it looks good to go

### CODE REVIEW

File: lib/tasks/job_fetch.rake [hunk 1]
```diff
@@ -1,15 +1,11 @@
 namespace :job_fetch do
   desc "Run the stack overflow fetcher"
 
-  task stackoverflow: :environment do
+  task all_jobs: :environment do
```
Comment (bethsecor) [inline lib/tasks/job_fetch.rake]: This is probably the easiest way to make sure we have one rake task for Heroku Scheduler.

**Comment by bethsecor** [comment]
This looks good Slota! :sparkles:

**Comment by rrgayhart** [comment]
What might be a better implementation long term here would be to have two distinct tasks that we can run for local testing purposes - and then one rake task that runs both tasks.

That way if one task stops working correctly, we can handle it quickly

File: lib/tasks/job_fetch.rake [hunk 2]
```diff
@@ -1,15 +1,11 @@
 namespace :job_fetch do
   desc "Run the stack overflow fetcher"
```
Comment (rrgayhart) [inline lib/tasks/job_fetch.rake]: The desc for the task should match what the task does - so this is slightly out of date now

**Pull request merged**